src/containers/Flow/Flow.tsx

```diff
@@ -20,8 +20,11 @@ export interface FlowProps {
   match: any;
 }
 
+const regex = /^[a-z]+(,[a-z]+)*$/g;
+
 const FormSchema = Yup.object().shape({
   name: Yup.string().required('Name is required.'),
+  keywords: Yup.string().matches(regex, 'Only lowercase are allowed.'),
```

Comment:
```suggestion
  keywords: Yup.string().matches(regex, 'Only lowercase are allowed.'),
```
Can we have a generic error message as currently if we press space then also we get the same error.

Reply:
Space is also not a valid input for this, it only allows comma separated lowercase keywords. We also have helper text for this.